MIP102/MIP102c2-Subproposals/MIP102c2-SP17.md

```diff
@@ -7,8 +7,9 @@ MIP102c2-SP#: 17
 MIP to be amended: MIP104, MIP106, MIP107, MIP108, MIP113
 Author(s): @IamMeeoh
 Contributors: @vigilant, @Bonapublica, @Palc
+General Edit or Article 1 Edit: Article 1 Edit
```

Comment:
We are not sure that this information will work as is presumably intended. The MIPs Portal does not appear to parse it when used elsewhere such as in https://mips.makerdao.com/mips/details/MIP102c2SP18#specification

If this information is desired to be displayed, we suggest a tag might be a better method until the portal UI can reflect adding this information as a separate element of the preamble

Reply:
That is correct. The Portal doesn't know what to do with this at this point in time. This field is supposed (so is my understanding) to make extra-sure authors are aware that they need to choose between these two alternatives. Tags are a good idea, though. I can do a mass tagging of all MIP102c2 subproposals later (without delaying the merging of this PR)

Reply:
On second thought, I'll add the tags for these now